README.md

```diff
@@ -107,7 +108,7 @@ You can access these pages on your computer using one of the following clients:
   - [tldr-python-client](https://github.com/tldr-pages/tldr-python-client):
     `pip install tldr` or `pacman -S tldr` on Arch Linux
   - [tldr.py](https://github.com/lord63/tldr.py):
-    `pip install tldr.py`
+    `pip install tldr.py` or `apt-get install tldr-py` on Ubuntu
```

Comment:
Does the `apt-get` command work on other Debian-based distributions? If so, it would be worth using `Debian-based distributions` here instead.
```suggestion
    `pip install tldr.py` or `apt-get install tldr-py` on Debian-based distributions
```

Reply:
That's true. I will commit your suggestions @sbrl.